manifests/vizier/db/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: vizier-db-secrets
+  namespace: katib
+data:
+  MYSQL_ROOT_PASSWORD: test
```

Comment:
I think we should take a "secure by default" approach and the value should be randomized, while its implementation can be a bit messy. Also it has to be initialized before the mysql pod starts up.
At least there should be an issue so that we don't forget to randomize before GA. ;-)

Reply:
IMHO editing manifest for the secret before running deploy.sh seems not much burden.

Reply:
To do that, users must be aware of the secret.yaml file. Randomization must happen automatically or the majority of deployments would be in an insecure state.

Reply:
I agree with your point "secure by default". How about dynamically generating manifests when executing `./scripts/build.sh`, or introduce a new script, e.g. `./scripts/generate-manifests.sh`? Other projects tend to have users do that sort of thing, such as `$ make manifests`. I think it's much better than doing it in `./scripts/deploy.sh`